runtime/oti/j9nonbuilder.h

```diff
@@ -5997,6 +5997,7 @@ typedef struct J9JavaVM {
 #if defined(J9VM_OPT_CRIU_SUPPORT)
 	omrthread_monitor_t delayedLockingOperationsMutex;
 #endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
+	UDATA asyncGetCallTraceUsed;
```

Comment:
Minor:  can we make this `BOOLEAN`

Reply:
I would prefer to move away from our fake types (in a possible move towards using only C++). I also want this to explicitly be a full-word field in case there are any unforseen problems because we're doing this in a signal handler.